press.md

```diff
@@ -5,6 +5,8 @@ css_id: press
 
 # Press
 
+* [The Drive.com-July 23,2019: Top OTA Expert Shows How State Actors Hack into Your Car and What Happens Next: ‘People Will Die’](https://www.thedrive.com/tech/29120/top-ota-expert-shows-how-state-actors-hack-into-your-car-and-what-happens-next-people-will-die?__s=fhb4zzuy2usv5bfnw9cb)
```

Comment:
It probably should be in a separate commit.  It's not related here.  

Also, the LWN article is good to cite also

Reply:
Ok. I can take that out and add it separately. It makes sense since T=that commit doesn't need to be reviewed.

Reply:
DONE